packages/core/src/lib/light_push/index.ts

```diff
@@ -49,7 +49,8 @@ class LightPush extends BaseProtocol implements ILightPush {
 
   constructor(libp2p: Libp2p, options?: ProtocolCreateOptions) {
     super(LightPushCodec, libp2p.components);
-    this.pubsubTopics = this.initializePubsubTopic(options?.shardInfo);
+    this.pubsubTopics =
```

Comment:
nit: seems repetitive, let's include into `this.initializePubsubTopic`

Reply:
I did it this way because we will be removing `options.pubsubTopics` in the future. This way, we just need to remove the `??` expression in this spot while keeping the function signature and behavior of `initializePubsubTopics` the same